lib/fluent/plugin/out_elasticsearch_data_stream.rb

```diff
@@ -142,7 +142,7 @@ def data_stream_exist?(datastream_name, host = nil)
         response = client(host).indices.get_data_stream(params)
         return (not response.is_a?(TRANSPORT_CLASS::Transport::Errors::NotFound))
       rescue TRANSPORT_CLASS::Transport::Errors::NotFound => e
-        log.info "Specified data stream does not exist. Will be created: <#{e}>"
+        log.info "Specified data stream does not exist. Will be created: <#{datastream_name}>"
```

Comment:
How about decrease log level for entire log and add the proposed line?
This should be more reasonable to suppress log warnings. What do you think?

Reply:
Noted. Let's change the log message. No need to further action.

Reply:
Ok thanks! Is there anything else I need to do before you can merge this?

Reply:
There is no action for you. I'm just late for merging this. Thank you for your work. Good catch. :smile:

Reply:
No worries, thank you!